### PR TITLE
 Fix `build.rs` when both -I & -L in present in oneline

### PR DESCRIPTION
**Pull request by NobodyXu**
and where the flag and value is separated by one or
more ascii whitespace (`\n`, `\t`, whitespace).

Signed-off-by: Jiahao XU <Jiahao_XU@outlook.com>

### CODE REVIEW

**Comment by NobodyXu** [comment]
@jmjoy Request review on this PR.

File: graphicsmagick-sys/build.rs [hunk 1]
```diff
@@ -35,19 +36,24 @@ fn new_graphicsmagick_config() -> anyhow::Result<GraphicsMagickConfig> {
...
+        let (flag, value) = if ["-I", "-L", "-l"].contains(&token) {
+            // Since token only contains the flag, the next element of it
+            // must contains the value.
+            (token, it.next().unwrap())
+        } else {
+            (&token[..2], &token[2..])
+        };
+
+        match flag {
+            "-I" => gmc.include_flags.push(format!("-I{value}")),
```
Comment (jmjoy) [inline graphicsmagick-sys/build.rs:53]: The `format!("-I{value}"` is the new version rust feature, please update the `Cargo.toml` `rust-version` and README label.

Comment (NobodyXu) [reply graphicsmagick-sys/build.rs]: I've replaced this with `format!("-I{}", value)`

Comment (jmjoy) [reply graphicsmagick-sys/build.rs]: :+1:

**Review by jmjoy — APPROVED** [review]

**Pull request merged**